packages/main/src/plugin/extension-loader.ts

```diff
@@ -429,6 +429,7 @@ export class ExtensionLoader {
   }
 
   async analyzeExtension(extensionPath: string, removable: boolean): Promise<AnalyzedExtension> {
+    extensionPath = await realpath(extensionPath);
```

Comment:
I would rename extensionPath of the parameter (to be sure we don't modify the parameter)